src/prost/tendermint.abci.rs

```diff
@@ -1,14 +1,14 @@
 /// Generated client implementations.
 #[cfg(feature = "client")]
-pub mod abci_application_client {
+pub mod abci_client {
```

Comment:
Is the tendermint package still used anywhere in the other protos?
It would be good to get rid of these generated files and replace with tendermint-rs. But maybe it's not a job for this PR.

Reply:
Good catch! We don't use them so I've updated the script to remove the generated file.